documentation/release_notes/release_notes_latest.markdown

```diff
@@ -21,6 +21,85 @@ canonical: "/puppetdb/latest/release_notes.html"
 
 ---
 
+## PuppetDB 6.13.0
+
+Released 20 October 2020
+
+### New features and improvements
+
+- PuppetDB can now log the AST and SQL corresponding to each incoming
+  query when requested by the
+  [log-queries](https://puppet.com/docs/puppetdb/latest/configure.html#log-queries)
+  configuration option.
+  ([PDB-4834](https://tickets.puppetlabs.com/browse/PDB-4834))
+
+- PuppetDB will only drop the oldest report or events partition during
+  the normal, periodic gc when there's more than one candidate. This
+  is intended to decrease the length of time we might block pdb
+  operations since with the current arrangement, the drop will attempt
+  to acquire an exclusive lock on the entire table (i.e. reports, not
+  just the partition), and so will block all subsequent access to that
+  table, until it finishes.
+  ([PDB-4901](https://tickets.puppetlabs.com/browse/PDB-4901))
```

Comment:
Suggest rewrite of note above: 
PuppetDB only drops the oldest report or events partition during the normal periodic gc when there is more than one candidate. This decreases the length of time PuppetDB blocks operations. Currently, the drop attempts to acquire an exclusive lock on the entire table (i.e. reports, not just the partition) and blocks all subsequent access to that table until it finishes.